processor/sumologicsyslogprocessor/testdata/sumologic_syslog_config.yaml

```
@@ -0,0 +1,16 @@
+receivers:
+  nop:
+
+exporters:
+  nop:
+
+processors:
+  sumologic_syslog:
+    facility_attr: testAttrName
+
+service:
+  pipelines:
+    traces:
```

```suggestion
    logs:
```

Wouldn't this make more sense ?